src/interfaces/store-interfaces/state-model-interface.ts

```diff
@@ -1,40 +1,51 @@
 import {
-  AccountInformationIF, AlterationFilingIF, CertifyIF, DefineCompanyIF, NameRequestIF, TombStoneIF, PeopleAndRolesIF,
-  ShareStructureIF, EffectiveDateTimeIF, IncorporationAgreementIF, BusinessInformationIF, IncorporationFilingIF,
+  AccountInformationIF, AlterationFilingIF, CertifyIF, NameRequestIF, TombStoneIF, PeopleAndRolesIF, ShareStructureIF,
+  EffectiveDateTimeIF, IncorporationAddressIf, IncorporationAgreementIF, BusinessInformationIF, IncorporationFilingIF,
   FilingDataIF, StaffPaymentStepIF, DetailIF, NameTranslationIF, BusinessSnapshotIF, DocumentDeliveryIF, FeesIF,
   NewAlterationIF
 } from '@/interfaces'
+import { ContactPointIF } from '@bcrs-shared-components/interfaces'
 
 /** Data object used internally only (not to/from API). */
 export interface StateModelIF {
   currentJsDate: Date
   tombstone: TombStoneIF
+  newAlteration: NewAlterationIF
   accountInformation: AccountInformationIF
+  businessContact: ContactPointIF
   businessInformation: BusinessInformationIF
   nameRequest: NameRequestIF
   nameTranslations: NameTranslationIF[]
   certifyState: CertifyIF
   documentDelivery: DocumentDeliveryIF
-  defineCompanyStep: DefineCompanyIF
+  officeAddresses: IncorporationAddressIf | {}
   peopleAndRolesStep: PeopleAndRolesIF
   shareStructureStep: ShareStructureIF
   incorporationAgreementStep: IncorporationAgreementIF
   effectiveDateTime: EffectiveDateTimeIF
   originalIA: IncorporationFilingIF
   originalAlteration: AlterationFilingIF
-  newAlteration: NewAlterationIF
-  originalSnapshot: BusinessSnapshotIF
+  businessSnapshot: BusinessSnapshotIF
   staffPaymentStep: StaffPaymentStepIF
   filingData: FilingDataIF
   detail: DetailIF
+  changedFlags: {
+    // FUTURE: move all flags here
+    defineCompanyStep: boolean
```

Comment:
I think it makes sense to keep the flags together.
On the other hand, I think we should not put data in specific step objects (ie, when the data is used by different filing types)... Note that I deleted DefineCompanyIF.

Reply:
This is a partial step into "flattening" our data structure so that it's filing-agnostic. In other words, the data should not be organized according to a specific filing type (since that makes a mess when a new filing needs data from a previous filing type).
^^ We will need to do this to Create UI when we start adding more filing types there.